$NetBSD: patch-Makefile,v 1.4 2015/10/20 12:13:33 fhajny Exp $

Add explicit NetBSD 64-bit target.
Skip unpacking, handled by pkgsrc.

--- Makefile.orig	2014-07-04 05:20:00.000000000 +0000
+++ Makefile
@@ -20,7 +20,7 @@ $(SNAPPY_ARCHIVE):
 	@mkdir -p $(@D)
 	curl -o$@ http://snappy.googlecode.com/files/snappy-$(VERSION).tar.gz
 
-$(SNAPPY_UNPACKED): $(SNAPPY_ARCHIVE)
+$(SNAPPY_UNPACKED):
 	tar xvfz $< -C $(TARGET)	
 	touch $@
 
@@ -97,6 +97,9 @@ linux32:
 freebsd64:
 	$(MAKE) native OS_NAME=FreeBSD OS_ARCH=amd64
 
+netbsd64:
+	$(MAKE) native OS_NAME=NetBSD OS_ARCH=amd64
+
 # for cross-compilation on Ubuntu, install the g++-arm-linux-gnueabi package
 linux-arm:
 	$(MAKE) native CROSS_PREFIX=arm-linux-gnueabi- OS_NAME=Linux OS_ARCH=arm
